src/static/publish/s3/put-files/index.js

```diff
@@ -50,10 +54,12 @@ module.exports = function putFiles (params, callback) {
 
           // Only upload if the file was modified since last upload
           // In theory we could use the ETag, but Amazon uses an unpublished chunk hashing algo
```

Comment:
Should we update this comment now that the code is using ETags?

Reply:
Hilarious that that comment is right above the code that is now using ETag and I didn't even notice it. 🙃